dts/bindings/can/can-controller.yaml

```diff
@@ -10,9 +10,12 @@ properties:
       Initial bitrate in bit/s.
   sample-point:
     type: int
-    required: true
     description: |
       Initial sample point in per mille (e.g. 875 equals 87.5%).
+
+      If this is unset (or if it is set to 0), the initial sample point will default to 75.0% for
```

Comment:
I would not write out the values here but rather something "automatically determined in accordance with guideline from the CiA".
But good Idea to make them optional!